### PR TITLE
Menu, little fixes

### PR DESCRIPTION
**Pull request by alexdp1996**

### CODE REVIEW

File: TaskManager.DTO/UserDTO.cs [hunk 1]
```diff
@@ -10,14 +10,20 @@ public class UserDTO
     {
         public string Id { get; set; }
 
+        [MaxLength(20)]
+        [MinLength(3)]
         [Required]
         public string Username { get; set; }
+
         [Required]
+        [MaxLength(30)]
```
Comment (katglin) [inline TaskManager.DTO/UserDTO.cs]: add [EmailAddress] attribute?

File: TaskManager.Web/Controllers/InviteController.cs [hunk 2]
```diff
@@ -103,14 +103,10 @@ private void UpdateInvitesNumber(string userName, int invitesNumber)
...
-                var userName = InviteService.RemoveInvite(teamId, inviteId);
+            var userName = InviteService.RemoveInvite(teamId, inviteId);
+            if  (!string.IsNullOrEmpty(userName))
                 UpdateInvitesNumber(userName, InviteService.GetInvitesNumber(userName));
-            }
-            catch
-            {
-            }
+
             return RedirectToAction("Index", new { teamId });
```
Comment (katglin) [inline TaskManager.Web/Controllers/InviteController.cs]: If we failed to delete an invite, shall we just redirect user without any explanation?

Comment (alexdp1996) [reply TaskManager.Web/Controllers/InviteController.cs]: user wont see that invite after all so it doesnt matter

File: TaskManager.DTO/UserDTO.cs (hunk 1 above)
Comment (alexdp1996) [reply TaskManager.DTO/UserDTO.cs]: added

**Pull request merged**